ceph.spec.in

```diff
@@ -3,6 +3,7 @@
 %bcond_with tests
 %bcond_without tcmalloc
 %bcond_without libs_compat
+%bcond_with librocksdb
```

Comment:
right, and this is exactly what is wanted. This behaviour is experimental behaviour.

It is not even enabled by default in autoconf. If this is default behaviour it should be default in the src build before it becomes default in the rpm, unless we want to encourage bad feeling toward some types of distributions even by association.